### PR TITLE
Clear selectedPosts when postType is changed

### PR DESCRIPTION
**Pull request by s3rgiosan**
This PR addresses issue #10

### CODE REVIEW

**Comment by s3rgiosan** [comment]
@psorensen for visibility

**Comment by s3rgiosan** [comment]
Hi @psorensen any news on this?

File: assets/js/admin/block-extension.js [hunk 1]
```diff
@@ -15,6 +16,29 @@ const BlockEdit = (props) => {
...
+
+			if (newAttributes.query && newAttributes.query.include) {
+				delete newAttributes.query.include;
+			}
+
+			newAttributes.selectedPosts = [];
+			newAttributes.query = {
+				...newAttributes.query,
+				orderBy: 'date',
+				perPage: newAttributes.query.perPage,
```
Comment (psorensen) [inline assets/js/admin/block-extension.js:33]:
```suggestion
				perPage: newAttributes.query.perPage,
				include: [],
```

Comment (psorensen) [reply assets/js/admin/block-extension.js:33]: This prevents a single post from showing by default when switching between post types.

**Review by psorensen — APPROVED** [review]
@s3rgiosan One suggestion here. Test it out and feel free to merge when you get a chance.

**Comment by s3rgiosan** [comment]
@psorensen Merged your suggestion. Thanks!

**Pull request merged**